### PR TITLE
V2

### PR DESCRIPTION
**Pull request by meliaj**
Self explanatory commit messages

### CODE REVIEW

File: full-v2.yml [hunk 1]
```diff
@@ -0,0 +1,2289 @@
...
+      phone_type:
+        type: string
+        x-nullable: true
+      relationship:
+        type: string
+        x-nullable: true
+      sis_id:
+        type: string
+        x-nullable: true
+      student:
```
Comment (beenakamath) [inline full-v2.yml]: Students should be an array on the v2.0 contacts object

File: full-v2.yml [hunk 2]
```diff
@@ -0,0 +1,2289 @@
...
+        type: string
+        x-validation: true
+      type:
```
Comment (beenakamath) [inline full-v2.yml]: Should we make the type and relationship enums? [values mappings: https://github.com/Clever/pipeline-data-quality/blob/master/valuemapper/contacttypes.yml and https://github.com/Clever/pipeline-data-quality/blob/master/valuemapper/contactrelationships.yml]

Possibly also phone_type.

File: main-v2.go [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"gopkg.in/v1/yaml"
+)
+
+func main() {
+
+	fullBytes, err := ioutil.ReadFile("./full-v1.yml")
```
[the remark this answers is not in the document]
Comment (meliaj) [reply main-v2.go]: Fixed in later commit

File: Makefile [hunk 4]
```diff
@@ -1,2 +1,5 @@
 generate-v1:
 	go run main-v1.go
+
+	generate-v2:
```
[the remark this answers is not in the document]
Comment (meliaj) [reply Makefile]: Fixed in later commit

File: full-v2.yml (hunk 1 above)
Comment (meliaj) [reply full-v2.yml]: Good catch

**Comment by meliaj** [comment]
Addressed comments

**Pull request merged**